carbon-design-system/carbon-addons-iot-react
# feat: adjust padding when using filters in table

Closes #3231 

**Summary**

This isn't a full 100% fix. The right-align items specifically were left out, because we don't currently change the alignment of the headers--only the column content when aligning right or center. Otherwise, the other items were addressed, I believe.

**Change List (commits, features, bugs, etc)**

- it adds padding to the edges and bottom of the filters
- adjusts the height of the header based on table row size
- adjusts padding to align column title with edge of content/placeholder in filters
- added borders between filter sections when using column groupings
- updated snapshots

**Acceptance Test (how to verify the PR)**

- confirm these changes match the styles in the original issue--minus the right-alignment specs

**Regression Test (how to make sure this PR doesn't break old functionality)**

- check RTL/LTR with various knobs and demos in the playground to make sure edge cases weren't missed



**Things to look for during review**
